engine/tests/phpunit/Elgg/TestCase.php

```diff
@@ -7,6 +7,10 @@
 use stdClass;
 use Zend\Mail\Transport\InMemory as InMemoryTransport;
 
+/**
+ * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
```

Comment:
Ahh, for 1.12 we'd have to put these in the phpunit.xml. Seems OK to do that though.